MdePkg/Include/Register/Intel/StmApi.h

```diff
@@ -52,10 +52,19 @@ typedef struct {
   ///
 } SOFTWARE_STM_HEADER;
 
+// MU_CHANGE - [START]: Add new CPU information header for STM
 typedef struct {
-  MSEG_HEADER            HwStmHdr;
-  SOFTWARE_STM_HEADER    SwStmHdr;
+  UINT32    NumberOfCpus;
+  /// Pad to take up 1KB of space
+  UINT8     Reserved[SIZE_1KB - sizeof (UINT32)];
+} CPU_INFORMATION_HEADER;
+
+typedef struct {
+  MSEG_HEADER               HwStmHdr;
+  CPU_INFORMATION_HEADER    CpuInfoHdr;

```

Comment:
I thought we are going to add this after the software header?

Reply:
`MSEG_HEADER` ends with `UINT8     Reserved[SIZE_2KB - 8 * sizeof (UINT32)];`, `SOFTWARE_STM_HEADER` is expected to immediately follow at that offset. This would have to come after it to make the change `STM_HEADER` backward compatible.